bootstrap-languages/p-diff-sync/rollup.config.hc-dna.js

```
@@ -6,18 +6,18 @@ import sveltePreprocess from "svelte-preprocess";
 import postcss from "rollup-plugin-postcss";
 import { string } from "rollup-plugin-string";
 import json from "@rollup/plugin-json";
-import dna from "@perspect3vism/rollup-plugin-dna";
+import { happ } from "@coasys/rollup-plugin-holochain";
 
 const production = !process.env.ROLLUP_WATCH;
 
 export default {
-  input: "dna.js",
+  input: "happ.js",
   external: [],
```

Biome (1.9.4)

[error] This property value named external is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named external


(lint/suspicious/noDuplicateObjectKeys)